spyder/plugins/outlineexplorer/widgets.py

```diff
@@ -159,6 +159,9 @@ def remove_from_tree_cache(tree_cache, line=None, item=None):
         for line, (_it, _level, _debug) in list(tree_cache.items()):
             if _it is item:
                 break
+    if line is None:
+        # Couldn't find the item

```

Comment:
Maybe it is better to not use contractions? Makes it more readable `Could not`

Reply:
To a native speaker, ``Could not`` sounds a bit odd in that context (since a native would almost always use the contraction there), but if it makes it easier to understand for non-native speakers (the vast majority of our developers and users) then I'm all for avoiding contractions.

Reply:
Thanks for the input, well as a non native, I think it is clearer to not contract :-|, but I don't know ;-)

Reply:
You're definitely correct; that's what most technical writing guides directed at writing for an highly international audience that I've read recommend.